### ISSUE TITLE
Settable data location

### ISSUE THREAD

**Issue by jmjwozniak** [ECP-CANDLE/Benchmarks#34]
Do not force big input data to reside inside a git clone.  This forces code and big data to reside on the same FS (I use a soft link to get around this), and is likely to trigger git mistakes.

**Comment by jmohdyusof** [comment]
Does this mean you want the 'Data' directory to reside outside the Benchmarks directory? Do we want a standard way to set the Data dir base address?

**Comment by jmjwozniak** [comment]
Yes, I think that should be possible.  For example, on some machines I want to clone the Benchmarks in my home directory (fully backed up, small quota) but keep the input data in some project scratch directory (data possibly lost or evicted, large quota).

**Comment by jmohdyusof** [comment]
Resurrecting this, since it is still relevant and I have been working with Tom on it. The CANDLE_DATA_DIR environment variable is starting to get to this, but we need more generally to implement the IO going to writeable filesystem locations, as needed by both e.g. Summit and singularity containers
